web/src/settings_profile_fields.js

```diff
@@ -581,19 +581,9 @@ function update_profile_fields_checkboxes() {
 
 function toggle_display_in_profile_summary_profile_field(e) {
```

Comment:
Before, we were toggling the value from the field, now we are checking the `checked` prop of the checkbox. While, it is not technically a toggle anymore, in all practicality it would always toggle, so I've kept the name same instead of changing the `toggle` to `update`.